src/main/minio.js

```diff
@@ -324,6 +324,9 @@ export class Client {
     if (!isString(region)) {
       throw new TypeError('region should be of type "string"')
     }
+    if ((returnResponse !== true) && (returnResponse !== false)) {
```

Comment:
Would `typeof returnResponse != 'boolean'` be better here?

Reply:
We have a functions isBoolean for this purpose.

Reply:
Ah yes, will use that